zebrad/src/components/tracing/component.rs

```diff
@@ -47,6 +69,7 @@ impl Tracing {
 
             let logger = FmtSubscriber::builder()
                 .with_ansi(use_color)
+                .with_writer(non_blocking)
```

Comment:
Do we need to add the writer to the `ErrorLayer` as well, or does it use the same writer as the `FmtSubscriber`?

Reply:
No need or ability to add the writer to the ErrorLayer, it uses the writer from the FmtSubscriber